packages/indexer/indexer/chain_mapper.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+from typing import Dict, List
+
+
+@dataclass
+class CosmosChain:
+    chain_id: str
+    min_block_height: int
+    apis: List["CosmosAPI"] = field(default_factory=list)
+    current_api_index: int = 0
+
+
+@dataclass
+class CosmosAPI:
+    url: str
+    archive: bool = False
+
+
+chain_mapping: List[CosmosChain] = [
+    CosmosChain(
+        chain_id="secret-4",
+        min_block_height=7284419,
+        apis=[
+            CosmosAPI(url="https://secret-4.api.trivium.network:1317"),
+        ],
+    ),
+    CosmosChain(
+        chain_id="jackal-1",
```

Comment:
We talked about this in Discord, but I think it is a bad design decision to jam all different chains into a single indexer.